### PR TITLE
Fabo/2224 persist session

### LINKED ISSUE #2224: Persist session
**Is your feature request related to a problem? Please describe.**

When I refresh I don't want to sign in again.

**Describe the solution you'd like**

My session is persisted in a cookie or the URL.

Persisting it in the URL makes this a very stateless change.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by faboweb**
Closes #2224 

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added entries in `PENDING.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by jbibla** [comment]
nice! should probably bump the coverage, otherwise it has my approval!

File: app/src/renderer/scripts/boot.js [hunk 1]
```diff
@@ -109,8 +109,12 @@ export const startApp = async (
     store.commit(`setRpcUrl`, urlParams.rpc)
   }
 
-  store.dispatch(`connect`)
   store.dispatch(`showInitialScreen`)
+  store.dispatch(`connect`)
+    // wait for connected as the check for session will sign in directly and query account data
+    .then(() => {
```
Comment (fedekunze) [inline app/src/renderer/scripts/boot.js]: why not `await` ?

Comment (faboweb) [reply app/src/renderer/scripts/boot.js]: because then it would await until connected, but we want to return the vue instance on line 119

**Comment by faboweb** [comment]
> Although I get way more already subscribed errors on the console than before

every time you log in again. We will fix #1970 together with #1858 I think.

**Pull request merged**